src/components/Profile/Profile.js

```diff
@@ -0,0 +1,62 @@
+import React from "react";
+
+import "./Profile.scss";
+import Avatar from "../Avatar/Avatar";
+import { Button } from "../button/Button";
+
+import { HeartIcon, CommentIcon, FavouriteIcon } from "../../Assets/Icons/Icons";
+
+export const ProfileTop = ({ image, authorName, followerNum }) => {
+  return (
+    <div className='profile profileTopWrap'>
+      <Avatar image={image} dotted={true} />
+      <h4 className='title'>{authorName || "John Doe"}</h4>
+      <Button label={`Follow ${followerNum || "1.2"}k`} />
```

Comment:
even the `k` must come with a prop, Think like what if there are only 2 followers. For example: With your code, the person having 2 followers will have 2K followers printed. 
And we don't do fake data